ansible/tasks/docker/docker-compose.yaml.j2

```
@@ -12,7 +12,7 @@ services:
       - RUST_LOG=essential=debug
     volumes:
       - "/home/ec2-user/gear-data/:/gear/"
-    command: gear-node --base-path /gear/  --prometheus-external {% if rpc is not defined %} --validator  --telemetry-url 'ws://telemetry-backend-shard.gear-tech.io:32001/submit 0' {% endif %} {% if rpc is defined %} --unsafe-ws-external --unsafe-rpc-external  --telemetry-url 'ws://telemetry-backend-shard.gear-tech.io:32001/submit 0' --rpc-methods Unsafe --rpc-cors all {% endif %} {% if bootnodeId is defined %} --bootnodes /ip4/{{ bootnode }}/tcp/30333/p2p/{{ bootnodeId }} {% endif %} {% if archive_node is defined and archive_node == "yes" %} --pruning archive {% endif %}
+    command: gear-node --base-path /gear/ --prometheus-external {% if rpc is not defined %} --validator --telemetry-url 'ws://telemetry-backend-shard.gear-tech.io:32001/submit 0' {% endif %} {% if rpc is defined and unsafe == "yes" %} --unsafe-ws-external --unsafe-rpc-external {% endif %} {% if rpc is defined and unsafe == "no" %} --ws-external --rpc-external {% endif %} {% if rpc is defined %} --telemetry-url 'ws://telemetry-backend-shard.gear-tech.io:32001/submit 0' --rpc-methods Unsafe --rpc-cors all {% endif %} {% if bootnodeId is defined %} --bootnodes /ip4/{{ bootnode }}/tcp/30333/p2p/{{ bootnodeId }} {% endif %} {% if archive_node is defined and archive_node == "yes" %} --pruning archive {% endif %}
```

I'd rewrite this command:

```suggestion
    command: >
      gear-node --base-path /gear/ --prometheus-external
      --telemetry-url 'ws://telemetry-backend-shard.gear-tech.io:32001/submit 0'
      {% if name is defined %} --name={{ name }} {% endif %}
      {% if rpc is not defined %} --validator {% endif %}
      {% if rpc is defined and unsafe == "yes" %} --unsafe-ws-external --unsafe-rpc-external {% endif %}
      {% if rpc is defined and unsafe == "no" %} --ws-external --rpc-external {% endif %}
      {% if rpc is defined %} --rpc-methods Unsafe --rpc-cors all {% endif %}
      {% if bootnodeId is defined %} --bootnodes /ip4/{{ bootnode }}/tcp/30333/p2p/{{ bootnodeId }} {% endif %}
      {% if archive_node is defined and archive_node == "yes" %} --pruning archive {% endif %}
```